AppHandling/Run-AlPipeline.ps1

```diff
@@ -2426,6 +2431,28 @@ if (!(Test-Path $destFolder -PathType Container)) {
 $testApps | ForEach-Object {
     Copy-Item -Path $_ -Destination $destFolder -Force
 }
+
+# Copy error logs to build artifact folder
+if($generateErrorLog) {
+    # Define destination folder for error logs
+    $destFolder = Join-Path $buildArtifactFolder "ErrorLogs"
+    if (!(Test-Path $destFolder -PathType Container)) {
+        New-Item $destFolder -ItemType Directory | Out-Null
+    }
+
+    # Copy error logs for all apps
+    @($apps, $testApps, $bcptTestApps) | ForEach-Object {

```

Comment:
Is it only apps that successfully compiled that are in $apps, $testapps and $bcptTestApps? If an app fails to compile shouldn't we still get the errorlog?

Reply:
That's a good point, but I'm not sure the log file is generated if the app fails compiling. I'll check.

Reply:
Done, now the file are copied always.